resources/views/landing_index.edge

```diff
@@ -70,7 +62,7 @@
           <p class="profile__url">URL:
             <button class="btn btn-link profile__url--btn" data-value="https://{{domainBeingAccessed}}" onClick="copy(this)">
               <span>https://{{ domainBeingAccessed }}</span><span
-                class="btn btn-copy"><i class="far fa-copy"></i></span></button>
+                class="btn btn-copy"><svg class="fa-icon"><use xlink:href="/static-assets/img/fa-regular.svg#copy"></use></svg></span></button>
```

Comment:
@johnnycho Can you confirm that calling `/static-assets/img/fa-regular.svg` for each `<use>` tag does not count as individual requests in the browser? Usually the SVG sprite file is called once at the top of the page and then the `<use>` element only references the ID like `#copy` . Ex: https://css-tricks.com/svg-sprites-use-better-icon-fonts/#inject-that-svg-at-the-top-of-the-document

Reply:
@johnnycho Please use only 1 SVG sprite file (I think there are 3 in this PR) and only include the icons that are being used if you're concerned about file size